vendor/mobify-js/1.1.1/base

```diff
@@ -0,0 +1 @@
+Subproject commit 9a3e6220ac32aadbd12187d7a8899fc3a60ec525
```

Comment:
Looks like we are point to Helen's commit. I feel like instead we should delete the existing 1.1.1 branch (since it never got used) and merge Helen's stuff into a 1.1.1 branch, and point this submodule to that.